src/__mocks__/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import countriesReducer from './countries';
+
```

Comment:
All the tests are failing. Kindly see the screenshots below and make sure all the tests are passing.